### PR TITLE
[BEAM-6548] Suggest mentioning users at the top of PULL_REQUEST template

### PR DESCRIPTION
**Pull request by lgajowy**
What I changed: 

 - added a welcome message (brings attention, is nice and kind)
 - moved on top
 - mentioned places where it's easy to find people working on the project (makes things easier to newcommers)

Any suggestions are welcome. Thanks!

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by kennknowles** [comment]
LGTM. I'll wait just a minute to see if anyone else from the mailing list has a comment.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 ------------------------
 
+Hello! Thanks for your contribution! Please mention someone in a comment (`@username`) to speed up the reviewing process.  See [Project Committee](https://projects.apache.org/committee.html?beam) page, or simply tag a suggested reviewer if you don't have anyone on your mind. 
+
```
Comment (echauchot) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would replace 

> or simply tag a suggested reviewer if you don't have anyone on your mind

>  or simply tag **one of the** suggested reviewers **in the github list** if you don't have anyone on your mind

for it to be even more clear

Comment (echauchot) [reply .github/PULL_REQUEST_TEMPLATE.md]: And maybe but this sentence in bold

**Comment by lgajowy** [comment]
@echauchot I applied some of your suggestions. However, I noticed that newcommers **do not** have the rights to choose suggested reviewers. This might be part of the problem - letting them choose would make things even easier for them. 

What do you all think about enabling this option? Is this possible?

**Comment by lgajowy** [comment]
zrzut ekranu 2019-02-1 o 12 46 48

Screenshot from my colleague for reference (no cogwheel next to the reviewers list). members of the project have it.

**Comment by kennknowles** [comment]
Unfortunately it is not possible as far as we know right now: https://issues.apache.org/jira/browse/INFRA-16701

**Comment by echauchot** [comment]
@lgajowy even if contributors that are not committers cannot assign a reviewer on the right hand side of the screen, maybe we can do as we did before this menu existed, i.e. agree on a tag format such as **R:@user** in a comment to pick up a reviewer. And if they don't have the proposed list, maybe we can advice them to run a command such as `git blame -f $file | cut -d' ' -f 2-4 | sort | uniq` to get the top contributor and tag him in the R tag.

**Comment by lgajowy** [comment]
@kennknowles hmm. Bummer - I didn't know that.

@echauchot do you suggest changing the current message then? Could you suggest what exactly should be changed? 

I guess we could suggest git blame somewhere: 

> See [Project Committee](https://projects.apache.org/committee.html?beam) or simply tag one of suggested reviewers **or find one (git blame)** if you don't have anyone on your mind.

**Comment by echauchot** [comment]
@lgajowy I would suggest but I don't want to be too picky :)

>  After creating the pull request please **_mention someone_** in a comment (R:`@username`) or pickup one in the proposed list on the right to speed up the reviewing process. If you don't know who to chose you can use `git blame` to get main contributor. See [Project Committee](https://projects.apache.org/committee.html?beam)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (robertwb) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think the point of this blurb is not (necessarily) to tell an author how to find a reviewer, but to make them aware that finding a reviewer is a good next step. 

My suggestion would be something along the lines of 

"After creating the pull request please, please [choose a reviewer](relevant contrib guide paragraph) and mention them in a comment (`@username`) to speed up the reviewing process, or simply tag **one of the** suggested reviewers **in the github list** if you don't have anyone on your mind. Mapping from the project committee list to a github id may be non-obvious. 

Should this be part of the checklist? 

(I don't want to bikeshed on this, anything is better than nothing, but that's my two cents.)

Comment (lgajowy) [reply .github/PULL_REQUEST_TEMPLATE.md]: > I think the point of this blurb is not (necessarily) to tell an author how to find a reviewer, but to make them aware that finding a reviewer is a good next step.

Fair point. Given that Beam Contribution Guite will be linked and shows the ways of finding the reviewer I think it's enough. 

> Should this be part of the checklist?

It should be, I guess. But it should be on the top of it to draw atention the most.

**Comment by lgajowy** [comment]
I agree with Robert that specifics should be explained in [Contribution Guide](https://beam.apache.org/contribute/#make-your-change). There already are some suggestions - I think we can work on them later. Meanwhile, I reduced the sentence to a bullet in the checklist (which seems sensible, because we don't want other bullets to be missed by contributors too).

**Comment by lgajowy** [comment]
Ok, squash + merging this. Thank you for all the suggestions.

**Pull request merged**